LayoutTests/imported/w3c/web-platform-tests/WebCryptoAPI/generateKey/failures.js

```diff
@@ -94,7 +94,7 @@ function run_test(algorithmNames) {
                 results.push({name: algorithmName, hash: "SHA-256", modulusLength: 1024, publicExponent: publicExponent});
             });
         } else if (algorithmName.toUpperCase().substring(0, 2) === "EC") {
-            ["P-512", "Curve25519"].forEach(function(curveName) {
+            ["P-512"].forEach(function(curveName) {
```

Comment:
Can you explain the reason for this change?

Reply:
Now that the curve is implemented there is no need to address it as a not implemented curve and do such checks